src/main/java/org/fogbowcloud/arrebol/ArrebolController.java

```diff
@@ -158,4 +192,41 @@ public TaskState getTaskState(String taskId) {
         //FIXME:
         return null;
     }
+
+    private void setWorkerCreator(String type) throws IOException {
+        switch (type) {
+            case DockerConstants.DOCKER_TYPE:
```

Comment:
also, it seems we have create this enum only to support creating this object. I'd prefer to parse the conf file instead

Reply:
This constant is used in two moments, one to load the worker and another when the image id is loaded.

Reply:
It would be better to add this to a class (maybe an enum?) named WorkerTypes. Then, WorkerTypes.RAW and WorkerTypes.DOCKER. Or even between, ExecDriver.DOCKER || RAW

Reply:
Done. I created an enum called WorkerType as you recommended